entity-management-web/src/main/java/eu/europeana/entitymanagement/batch/processor/EntityDereferenceProcessor.java

```diff
@@ -101,4 +117,11 @@ private boolean isEuropeanaProxyUpdated(EntityRecord entityRecord){
   private boolean metisResponseMatchesExternalProxy(EntityRecord entityRecord, Entity metisResponse){
       return entityComparator.compare(entityRecord.getExternalProxy().getEntity(), metisResponse) == 0;
     }
+  
+  private void validateConstraints(Entity entity) throws EntityValidationException  {
```

Comment:
the minimal validation must not be integrated into the dereferencing step, but as own functionality in the EntityUpdateProcessor see step 4 in the specs https://docs.google.com/document/d/16k9PcCMFwl2LXjnnzotZRPc-QqM-Ar1D0VELHt4t_hA/edit#

Reply:
Improved.